scripts/create_django_issue.py

```diff
@@ -0,0 +1,290 @@
+
+    Only keeps track on (major, minor) versions, excluding patches and pre-releases.
+    """
+
+    major: int
+    minor: int
+
+    def __str__(self) -> str:
+        """To render as string."""
+        return f"{self.major}.{self.minor}"
+
+    @classmethod
+    def parse(cls, version_str: str) -> DjVersion:
+        """Parse interesting values from the version string."""
+        major, minor, *_ = version_str.split(".")
+        return cls(major=int(major), minor=int(minor))
+
+
+def get_package_info(package: str) -> dict:
+    """Get package metadata using PyPI API."""
+    # "django" converts to "Django" on redirect
+    r = requests.get(f"https://pypi.org/pypi/{package}/json", allow_redirects=True)
+    if not r.ok:
+        print(f"Couldn't find package: {package}")
+        sys.exit(1)
+    return r.json()
+
+
+def get_django_versions() -> Iterable[DjVersion]:
+    """List all django versions."""
+    django_package_info: dict[str, Any] = get_package_info("django")
+    releases = django_package_info["releases"].keys()
+    for release_str in releases:
+        if release_str.replace(".", "").isdigit():
+            # Exclude pre-releases with non-numeric characters in version
+            yield DjVersion.parse(release_str)
+
+
+def get_name_and_version(requirements_line: str) -> tuple[str, ...]:
+    """Get the name a version of a package from a line in the requirement file."""
+    full_name, version = requirements_line.split(" ", 1)[0].split("==")
+    name_without_extras = full_name.split("[", 1)[0]
+    return name_without_extras, version
+
+
+def get_all_latest_django_versions() -> tuple[DjVersion, list[DjVersion]]:
+    """
+    Grabs all Django versions that are worthy of a GitHub issue.
+
+    Depends on Django versions having higher major version or minor version.
+    """
+    print("Fetching all Django versions from PyPI")
+    base_txt = REQUIREMENTS_DIR / "base.txt"
+    with base_txt.open() as f:
+        for line in f.readlines():
+            if "django==" in line:
+                break
+        else:
+            print(f"django not found in {base_txt}")  # Huh...?
+            sys.exit(1)
+
+    # Begin parsing and verification
+    _, current_version_str = get_name_and_version(line)
+    # Get a tuple of (major, minor) - ignoring patch version
+    current_minor_version = DjVersion.parse(current_version_str)
+    newer_versions: set[DjVersion] = set()
+    for django_version in get_django_versions():
+        if django_version > current_minor_version:
+            newer_versions.add(django_version)
+
+    return current_minor_version, sorted(newer_versions, reverse=True)
+
+
+_TABLE_HEADER = """
+
+## {file}.txt
+
+| Name | Version in Master | {dj_version} Compatible Version | OK |
+| ---- | :---------------: | :-----------------------------: | :-: |
+"""
+VITAL_BUT_UNKNOWN = [
+    "django-environ",  # not updated often
+    "pylint-django",  # classifier not included in setup.py
```

Comment:
I think we can remove this now: https://github.com/PyCQA/pylint-django/pull/337

Reply:
Though they haven't merged it yet and it'll only work once they public the package I believe... so I think we may need to hold off on removing this.